src/text-editor.coffee

```diff
@@ -677,7 +677,7 @@ class TextEditor extends Model
   # this editor.
   shouldPromptToSave: ({windowCloseRequested}={}) ->
     if windowCloseRequested
-      @isModified()
+      false
```

Comment:
Just tested with Notepad++ and it doesn't prompt to save on close.  I really don't see any downside to this because even if someone closed Atom accidentally, they can just open it up again?